src/stickyConfig.ts

```diff
@@ -9,16 +9,18 @@ const stickyConfigDisklet = makeReactNativeDisklet()
 
 const stickyDistribution = {
   swipeLastUsp: 0.5,
-  createAccountType: 0.1
+  createAccountType: 0.1,
+  legacyLanding: 1
```

Comment:
`legacyLanding: 1` makes this a noop for now?

Reply:
Debug changes, shouldn't have made it in